static/styles/lib/base.scss

```diff
@@ -3,7 +3,8 @@
 
 @import "./tether/tether.min.css";
 @import "./bootstrap/scss/bootstrap-flex";
-@import "./fontawesome/font-awesome";
+$fa-font-path: "/fonts/fontawesome";
+@import "../../../node_modules/font-awesome/scss/font-awesome.scss";
```

Comment:
hast du versucht es über webpack direkt zu importieren?
damit fällt dann auch node_modules weg.

Reply:
https://stackoverflow.com/questions/33649761/how-do-i-load-font-awesome-using-scss-sass-in-webpack-using-relative-paths
evtl.

Reply:
Jup, hast du Recht. Hab ich nicht drüber nachgedacht. Geht ja ganz easy.